### PR TITLE
some improvements suggested by davehunt

### PR DESCRIPTION
**Pull request by stupchiy**
Improvements to test localization options more properly

### CODE REVIEW

File: tests/test_file.py [hunk 1]
```diff
@@ -97,15 +96,16 @@ def test_robot_txt_present_on_site(self, mozwebqa):
 
     def test_request_ends_with_slash(self, mozwebqa):
         # BaseUrl Should be added without language in path for proper tests of locale (e.g. http://mozilla.org/firefox)
-        lang = 'en-US'
+        lang = ['en-US', 'de']
```
Comment (stephendonner) [inline tests/test_file.py]: We might not always have a German locale for a project, but en-US is almost always guaranteed; can we just leave this dynamic, for now?

e.g. can we grab the locales from the footer select pulldown and check that they show up in the URL field?

Comment (stupchiy) [inline tests/test_file.py]: According to this one
I have done that due to @davehunt suggestion to have tests using the accept-language header to determine the locale and this test (test_request_ends_with_slash) is running on background using urllib
For grabbing from footer select pulldown I have created this one - (test_locale) https://github.com/stupchiy/marketing-project-template/blob/fixes/tests/test_file.py#L29 , which is using WebDriver
So let me know if you want to remove some test or you want to do them both different things or any other suggestions... Will change it when needed

Comment (stephendonner) [inline tests/test_file.py]: Ah, ok.  Thanks for reminding me.  Let's just leave 'de' off, as we're not guaranteed a German locale, and we're covering the dynamic portions in test_locale, as you've also mentioned.

Thanks!

Comment (davehunt) [inline tests/test_file.py]: Without 'de' this is a list of one. Is that okay? Also, I'm not entirely clear how this relates to my suggestion but perhaps I don't have the full context here...

Comment (stephendonner) [inline tests/test_file.py]: Yes, this is fine.  We're never guaranteed a locale other than /en-US/, off the bat, so the test will fail until we have other localized versions trickle in (which we might never have).  The point is to test that the baseURL redirects, with its locale code, in place, like so:

## 

[15:42:37.598] GET http://www.mozilla.org/firefoxlive/ [HTTP/1.1 301 Moved Permanently 52ms]
[15:42:37.712] GET http://www.mozilla.org/en-US/firefoxlive/ [HTTP/1.1 200 OK 20ms]

Comment (davehunt) [inline tests/test_file.py]: If that's the case, we probably don't need the loop.

**Comment by stephendonner** [comment]
Thanks, Sergey (as always!)

**Pull request merged**